nvchecker/archlinux-proaudio.toml

```diff
@@ -11,6 +11,12 @@ include_regex = '^r(\d+)_(\d+)_(\d+)$'
 from_pattern = '^r(\d+)_(\d+)_(\d+)$'
 to_pattern = '\1.\2.\3'
 
+[jalv-select]
```

Comment:
This should go before jamulus, by lexical sorting ;)

Reply:
Fixed with 21bd0a6.